spine/sender.go

```diff
@@ -34,17 +32,14 @@ type Sender interface {
 }
 
 type SenderImpl struct {
-	msgNum uint64 // 64bit values need to be defined on top of the struct to make atomic commands work on 32bit systems
```

Comment:
This comment is done by myself. See https://github.com/evcc-io/eebus/commit/1829ea71928f037fdb0900ded39887aa42569c6a